### ISSUE TITLE
Update for shards 0.7.0

### ISSUE THREAD

**Issue by meew0** [ysbaddaden/scrypt-crystal#1]
The latest shards release broke the installation script by running `postinstall` scripts from the root of the dependency instead of the `src` folder. This can be reproduced by making a minimal `shard.yml`:

```
name: test-project
version: 0.1.0

dependencies:
  scrypt:
    github: ysbaddaden/scrypt-crystal
    branch: master
```
and then running `shards`, giving the following error:

```
Updating https://github.com/ysbaddaden/scrypt-crystal.git
Installing scrypt (master)
Postinstall cd ext && make
Failed cd ext && make:
/bin/sh: line 1: cd: ext: No such file or directory
```

**Comment by ysbaddaden** [comment]
This should be fixed in [v0.2.0](https://github.com/ysbaddaden/scrypt-crystal/releases/tag/v0.2.0).

**Comment by meew0** [comment]
Works perfectly, thanks 👍
